lib/kafo/puppet_command.rb

```diff
@@ -38,8 +38,11 @@ def self.search_puppet_path(bin_name)
       # Find the location of the puppet executable and use that to
       # determine the path of all executables
       bin_path = (::ENV['PATH'].split(File::PATH_SEPARATOR) + ['/opt/puppetlabs/bin']).find do |path|
```

Comment:
I still have doubts about this line. In production we actually want to prefer the last value over `$PATH` but in dev we don't. However, I think it's OK. The user with a non-trivial wrapper script could fix it and that's probably ok.